docs/pages/access-controls/access-request-plugins/access-monitoring-rules.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Routing Access Requests to Access Plugins
+description: How to set up Teleport's Access Monitoring Rules
+---
+
+With Teleport's Access Monitoring Rules Access Request notifications
+can be routed to plugins based on based on several conditions.
+For example, if you wish to notify different recipients e.g Slack channels,
+based on what roles or resources are being accessed, or based on what user
+is making the request.
+
+Currently only the Slack access plugin is supported.
+
+## Prerequisites
+
+- A managed Teleport Enterprise account.
+
+- The Enterprise `tctl` admin tool and `tsh` client tool version >= (=teleport.version=).
+
+  You can verify the tools you have installed by running the following commands:
+
+  ```code
+  $ tctl version
+  # Teleport Enterprise v(=teleport.version=) go(=teleport.golang=)
+  
+  $ tsh version
+  # Teleport v(=teleport.version=) go(=teleport.golang=)
+  ```
+
+  You can download these tools by following the appropriate [Installation 
+  instructions](../../installation.mdx#linux) for your environment and Teleport edition.
+
+- At least one of the Teleport Access Plugins that support using being set up.
+
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/2. Create an Access Monitoring Rule
+
+Here is an example Access Monitoring Rule. This rule will notify 'your_slack_channel'
+via the Slack plugin if an Access Request containing the role 'your_role_name' is
+made.
+
+```yaml
+kind: access_monitoring_rule
+version: v1
+metadata:
+  name: your-plugin-name
+spec:
+  subjects: ['access_request']
+  condition: 'access_request.spec.roles.contains("your_role_name")'
+  notification:
+    name: 'slack'
+    recipients: ['your_slack_channel']
+```
+
+Above is an example of an Access Monitoring Rule that routes incoming Access Requests
+that are requesting the role "your_role_name"to the slack channel "your_slack_channel".
+
+The above example can be created using `tctl create  your-file-name.yaml`
+from the command line.
+
+Or via the webUI by navigating to the Access Request page and clicking on
+`View Notification Routing Rules`.
+
+Multiple recipients can be specified in an Access Monitoring Rule.
+The condition field of the is set to a predicate expression defining the conditions
```

Comment:
Should we also add a link to [predicate language](https://goteleport.com/docs/reference/predicate-language/) here?
```suggestion
The condition field of the rule is set to a predicate expression defining the conditions
```